core/src/components/modal/modal.tsx

```diff
@@ -963,6 +975,149 @@ export class Modal implements ComponentInterface, OverlayInterface {
     }
   };
 
+  private initViewTransitionListener() {
+    // Only enable for iOS card modals when no custom animations are provided
+    if (getIonMode(this) !== 'ios' || !this.presentingElement || this.enterAnimation || this.leaveAnimation) {
+      return;
+    }
+
+    // Set initial view state
+    this.currentViewIsPortrait = window.innerWidth < 768;
+
+    // Create debounced resize handler
+    this.resizeListener = () => {
```

Comment:
Why did we go with this over:

```
@Listen('resize', { target: 'window' })
```

See content & col components

Reply:
Good question, let me fix it!